File: src/com/bean/Supervisor.java
package com.bean;

public class Supervisor {
	private int userid;
	private String password;
	private String sitename;
	public Supervisor(int userid, String password, String sitename) {
		super();
		this.userid = userid;
		this.password = password;
		this.sitename = sitename;
	}
	public int getUserid() {
		return userid;
	}
	public void setUserid(int userid) {
		this.userid = userid;
	}
	public String getPassword() {
		return password;
	}
	public void setPassword(String password) {
		this.password = password;
	}
	public String getSitename() {
		return sitename;
	}
	public void setSitename(String sitename) {
		this.sitename = sitename;
	}
	@Override
	public String toString() {
		return "Supervisor [userid=" + userid + ", password=" + password + ", sitename=" + sitename + "]";
	}

}

File: src/com/bean/Supervisor.go
package bean

import (
	"fmt"
)

type Supervisor struct {
	userid   int
	password string
	sitename string
}

func CreateSupervisor(userid int, password string, sitename string) Supervisor {
	return Supervisor{userid: userid, password: password, sitename: sitename}
}
func (s Supervisor) getUserid() int {
	return s.userid
}
func (s Supervisor) setUserid(userid int) {
	s.userid = userid
}
func (s Supervisor) getPassword() string {
	return s.password
}
func (s Supervisor) setPassword(password string) {
	s.password = password
}
func (s Supervisor) getSitename() string {
	return s.sitename
}
func (s Supervisor) setSitename(sitename string) {
	s.sitename = sitename
}
func (s Supervisor) String() string {
	return fmt.Sprintf("Supervisor [userid=%d, password=%s ,sitename=%s]", s.userid, s.password, s.sitename)
}
